### PR TITLE
coords.chandra added error checking

### PR DESCRIPTION
**Pull request by kglotfelty**
This adds some error checking to catch case like #236 where keywords missing, most like due to merged products.

I'm sure there could be more error checking done, but this is a lot better than before.

### CODE REVIEW

File: ciao-4.11/contrib/lib/python3.5/site-packages/coords/chandra.py [hunk 1]
```diff
@@ -44,6 +48,9 @@ def _setup_sim( keyword_list ):
     if all( [x in keyword_list for x in ["SIM_X", "SIM_Y", "SIM_Z"] ]):
         sim = [ keyword_list["SIM_X"], keyword_list["SIM_Y"], keyword_list["SIM_Z"] ]
     else:
+        logger.verbose0("WARNING: Missing one or more of the following keywords: "+
```
Comment (DougBurke) [inline ciao-4.11/contrib/lib/python3.5/site-packages/coords/chandra.py]: This could just use the @v0@ symbol you created above :-)

**Comment by DougBurke** [comment]
Can an entry be added to the changelog for this, mainly for advertising purposes?

**Comment by kglotfelty** [comment]
changelog updated in  10f5e7e4a221d8f

**Comment by DougBurke** [comment]
I rebased this to avoid the merges.

**Pull request merged**